src/rng/backends/rocrand/rocrand_helper.hpp

```diff
@@ -64,6 +64,8 @@
  */
 #ifndef _MKL_RNG_ROCRAND_HELPER_HPP_
 #define _MKL_RNG_ROCRAND_HELPER_HPP_
+// #include <hip/hip_runtime.h>
```

Comment:
did #include <hip/hip_runtime.h> commented on purpose?

Reply:
Yes, it was commented on purpose during testing. We've pushed a change to this file to remove it now.